app/services/trending_media_service.rb

```diff
@@ -1,13 +1,13 @@
 class TrendingMediaService 
 
   def self.details
-    response = conn.get("/trending/all/day")
+    response = conn.get("/3/trending/all/day")
```

Comment:
With this correction some stubs are out of date, I'll get that fixed in a smaller PR I'll be pushing up shortly though